client/src/components/ResourcesGrid.jsx

```diff
@@ -6,19 +6,37 @@ import ResourcePreview from './ResourcePreview';
 import '../css/ResourcesGrid.css';
 
 function ResourcesGrid(props) {
-  const { filteredResources, savedResources, updateSaved } = props;
+  const {
+    filteredResources,
+    savedResources,
+    updateSaved,
+    resourceCount,
+    updatePagination,
+    pageSize,
+    page,
+  } = props;
+
   const cards = (
     <List
       grid={{
         gutter: [32, 16],
         xs: 1,
-        sm: 2,
-        md: 4,
-        lg: 4,
-        xl: 6,
+        sm: 3,
+        md: 3,
+        lg: 3,
+        xl: 3,
         xxl: 3,
       }}
       dataSource={filteredResources}
+      pagination={{
+        pageSizeOptions: ['6', '15', '30'],
```

Comment:
(said this before last dev night)
Is it possible to change this to ints so we don't need the parseInts in the mongodb stuff? No big deal if you can't

Reply:
also remember fix the linting (run `yarn format`) and conflicts

Reply:
thanks Luciana! Also sadly I don't think I can change it to ints because antd wants a string array.